source/base/utilities.cc

```diff
@@ -85,12 +85,23 @@ namespace Utilities
                   << "Can't convert the string " << arg1
                   << " to the desired type");
 
+
+  std::string
+  dealii_version_string ()
+  {
+    return DEAL_II_PACKAGE_NAME " version " DEAL_II_PACKAGE_VERSION;
```

Comment:
I would have liked to have the git revision in this information but as discussed in #5515 we have reasons not to do that, so that's fine.